addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CrossSiteScriptingScanRule.java

```diff
@@ -684,6 +691,8 @@ public void scan(HttpMessage msg, String param, String value) {
                                                             .getResponseHeader()
                                                             .getHeader(HttpHeader.CONTENT_TYPE),
                                                     "json")) {
+                                                /* avoid sending a null byte to ZAP UI: */        
+                                                if (scriptAlert == GENERIC_NULL_BYTE_SCRIPT_ALERT) scriptAlert = GENERIC_SCRIPT_ALERT;
```

Comment:
This should use curly braces for consistency

Reply:
Hello, thank you for you quick feedback. I addressed all your points.